docs/faq/how-we-calculate-cost.mdx

```diff
@@ -27,6 +27,6 @@ In the case of Anthropic requests, there is no supported method for calculating
 
 ### Developer
 
-For a detailed look at our pricing table, please follow this link: [https://github.com/Helicone/helicone/blob/main/web/lib/sql/constants.ts#L4-L17](https://github.com/Helicone/helicone/blob/main/web/lib/sql/constants.ts#L4-L17)
```

Comment:
The term "our pricing table" is confusing - is it how much Helicone charges, or how  it estimates LLM provider pricing? I tried to make it more clear that its the latter